scripts/src/primazactl/cmd/join/parser.py

```diff
@@ -122,41 +129,77 @@ def add_args_join(parser: argparse.ArgumentParser):
         required=False,
         help=f"tenant to use for join. Default: \
             {DEFAULT_TENANT}",
-        default=DEFAULT_TENANT)
+        default=None)
+
+    # options
+    parser.add_argument(
+        "-y", "--dry-run",
+        dest="dry_run",
+        type=str,
+        required=False,
+        choices=settings.DRY_RUN_CHOICES,
+        default=settings.DRY_RUN_NONE,
+        help=f"Set for dry run (default: {settings.DRY_RUN_NONE})")
+
+    parser.add_argument(
+        "-o", "--output",
+        dest="output_type",
+        type=str,
+        required=False,
+        choices=settings.OUTPUT_CHOICES,
+        default=settings.OUTPUT_NONE,
+        help="Set to get output of resources which are created "
+             f"(default: {settings.OUTPUT_NONE}).")
 
 
 def join_cluster(args):
 
     try:
+
+        if not args.options_file:
+            if not args.environment and args.cluster_environment:
```

Comment:
second `if` seems wrong to me, also nesting can be reduced

```suggestion
        if not args.options_file and not ( args.environment and args.cluster_environment ):
```

Reply:
OK.